Functions/Gherkin.ps1

```diff
@@ -218,6 +222,8 @@ function Invoke-Gherkin {
 
         [Pester.OutputTypes]$Show = 'All',
 
+        [Switch]$HideStepData,
```

Comment:
@Jaykul: Just wanted to check, based on my previous comment, do you think it's a good idea to definitely add the option to disable the displaying of DocStrings/DataTables in the console to the `OutputType` enum? I'd like to wrap this up.

@nohwnd Do you have any preference as to how this should be handled, especially given that RSpec style tests don't make use of DocStrings nor DataTables (to my knowledge&mdash;I did see something called _implicit docstrings_, but that also implied the use of an extension to RSpec, so I wouldn't think this applies to Pester)?